### PR TITLE
remove Manifest

### PR DESCRIPTION
**Pull request by CarloLucibello**
let's be free of the Manifest

### CODE REVIEW

**Review by mcabbott — DISMISSED** [review]
bors r+

**Comment by mcabbott** [comment]
Failure might be new to this PR. It involves
```
[4] ctc_alpha(ŷ::CuArray{Float64, 2}, y::Vector{Int64})
  @ Flux.Losses /var/lib/buildkite-agent/builds/rtx4000-gpuci4-julia-csail-mit-edu/julialang/flux-dot-jl/src/losses/ctc-gpu.jl:211
```
at `/test/ctc-gpu.jl:26`. Looks odd to call this with a Vector & a CuArray, but seemingly worked before? Paging @maetshju 
```
@testset "ctc-gpu" begin
  x = rand(10, 50)
  y = rand(1:9, 30)
  x_cu = CuArray(x)
  g1 = gradient(ctc_loss, x_cu, y)[1]
```

**Comment by CarloLucibello** [comment]
yeah I'm seeing the same failure in #1516

**Comment by mcabbott** [comment]
Oh right. Slightly different stack trace but this line appears. CUDA v3.2.1 & NNlibCUDA v0.1.3, which is what the manifest there specifies.

**Comment by CarloLucibello** [comment]
Even removing the ctc test I see those errors everywhere. cc @maleadt

**Comment by DhairyaLGandhi** [comment]
Errors are due to be cuda and are tracked in #1648

**Comment by mcabbott** [comment]
Ok, I'll update the title

**Comment by CarloLucibello** [comment]
> Even removing the ctc test I see those errors everywhere. cc @maleadt

actually, that's wrong, there was a problem with my cuda setup. 

I fixed the `ctc_loss`, it shouldn't be doing scalar indexing now and tests should pass

File: test/outputsize.jl [hunk 1]
```diff
@@ -10,6 +10,7 @@
   @test outputsize(m, (10,); padbatch=true) == (2, 1)
   @test outputsize(m, (10, 30)) == (2, 30)
 
+  @info "Don't mind the following error, it's for testing purpose."
```
Comment (mcabbott) [inline test/outputsize.jl:13]: 👍

**Comment by CarloLucibello** [comment]
@mcabbott need an approval again

**Review by mcabbott — APPROVED** [review]
bors r+

**Pull request merged**